astro/src/content/docs/customize/email-and-messages/message-templates.mdx

```diff
@@ -53,9 +53,48 @@ Most templates will contain the User object as returned on the Retrieve User API
 
 Below you will find each stock template that FusionAuth ships for reference. The available replacement values will be outlined below for each template.
 
+## Default Forgot Password
+
+```
+[#setting url_escaping_charset="UTF-8"]
+To change your password click on the following link.
+
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+
+${url}
+
+- FusionAuth Admin
+```
+
+### Replacement Variables
+
+<APIBlock>
+  <APIField name="application" type="Application">
+    The Application object, see the [Application API](/docs/apis/applications) for field definitions.
+  </APIField>
+  <APIField name="changePasswordId" type="String">
+    The change password Id that must be included as a path segment in the `/password/change` link.
```

Comment:
I think this is the first mention of the `/password/change` endpoint here. It might be good to say what that is and/or link to its doc.

Reply:
Documentation is scant, but I linked to the template variables page.